tests/glsl-intrinsic/compute-derivative/intrinsic-derivative-function-in-compute.slang

```diff
@@ -10,12 +10,12 @@
 // CHECK_GLSL_LINEAR_C: layout(derivative_group_linearNV)
 
 //TEST:SIMPLE(filecheck=CHECK_SPV_QUAD_C): -allow-glsl -stage compute -entry computeMain -target spirv -DQUAD -DCOMPUTE
-// CHECK_SPV_QUAD_C: DerivativeGroupQuadsNV
-// CHECK_SPV_QUAD_C: "SPV_NV_compute_shader_derivatives"
+// CHECK_SPV_QUAD_C-COUNT-1: DerivativeGroupQuadsNV
+// CHECK_SPV_QUAD_C-COUNT-1: "SPV_NV_compute_shader_derivatives"
```

Comment:
I don't think `-COUNT-1` will do what you want.
You will need to do,
```
// CHECK_SPV_QUAD_C: DerivativeGroupQuadsNV
// CHECK_SPV_QUAD_C-NOT: DerivativeGroupQuadsNV
// CHECK_SPV_QUAD_C: "SPV_NV_compute_shader_derivatives"
// CHECK_SPV_QUAD_C-NOT: "SPV_NV_compute_shader_derivatives"
```

Reply:
I checked-- You are correct.
Thank-you for catching this.